### ISSUE TITLE
🐛 [BUG] - [BLISS OS 15 and above] [WIFI] WP3 or WP2/WP3 connection issue

### ISSUE THREAD

**Issue by falconslag** [BlissRoms-x86/support#63]
### Description

Hello,

I cannot connect to my Wifi which is configured to "WPA2/WPA3 Personal" on Bliss OS 15 or above.
But it works correctly when I connect through my smartphone using the mobile access point.
It also works correctly with Bliss OS 14.
If I reconfigure my Wifi to WPA2 Personal, it also works.

I see my Wifi in the list, I can select it, enter the key. Then it tries to connect and fails.

### Reproduction steps

```bash
1. Go to Wifi
2. Click on 'My Wifi Network Name'
3. Enter the Key
4. and => Connection Failure
```

### Screenshots

```bash
![DESCRIPTION](LINK.png)
```

### Category

Other

### OS Version

15.x, 16.x

**Comment by tankerkiller125** [comment]
Can confirm issue with WPA2/WPA3 mode, however WPA3 standalone has no issues in testing on my device.

**Comment by electrikjesus** [comment]
Can you please update with the wireless hardware involved? We need to know
what driver is loaded for it as well. Ifconfig or lsmod will show that info

**Comment by tankerkiller125** [comment]
Looks like (if I'm reading this correctly)

`cfg80211 958464 4 wl,iwlmvm,mac80211,iwlwifi` is the line you wanted.

If not I'll just paste in the whole lsmod results.

**Comment by electrikjesus** [comment]
You should be able to pipe lsmod to a file and upload that to the issue
```
lsmod > lsmod.txt
```

**Comment by tankerkiller125** [comment]
Sorry it took so long, been busy. Attached is the lsmod.txt file from the device I've been testing with.

[lsmod.txt](https://github.com/user-attachments/files/16923584/lsmod.txt)

**Comment by arianniroumand** [comment]
Yes i have same problem in wpa2 and wpa3 but when i shaire wifi with no password it can conect

**Comment by Sai-On-Git** [comment]
i have same issue with same versions of bliss os but there is Some Difference i can connect to my network by adding it to networks list but after sometime this issue comes up again and even after device wakes up from sleep this issue repeates it is very inconvinient to use the device with these problems
